src/components/vm/consoles/vnc.tsx

```diff
@@ -616,7 +617,11 @@ export const VncInactive = ({
         <>
             <EmptyState>
                 <EmptyStateBody>
-                    {_("Start the virtual machine to access the console")}
+                    <TextWithAction
+                        text={_("$0 the virtual machine to access the console")}
```

Comment:
I'm a bit sceptical that this translates well. E.g. think in German, the button action would be called "Starten" or "Stoppen erzwingen", and now imagine putting that into "Die Maschine $0 um auf die Konsole zuzugreifen". "Starten" would look weird, but not *too* bad, but the second is just gross.

As $0 is always a standalone phrase, please rather just put the "Start" into this string instead of interpolation.